Improve SEO metadata across all docs pages (#397)
powersync-ja/powersync-docs: client-sdks/orms/kotlin/sqldelight.mdx

---
title: "SQLDelight (Beta)"

> description: "Use SQLDelight with PowerSync's Kotlin SDK for type-safe SQL statements and auto-updating Flows, with known limitations and setup steps."

Reviewer: ```suggestion
description: "Learn how to use SQLDelight with PowerSync's Kotlin SDK."
```
